doc/man/8/ceph.rst

```diff
@@ -248,6 +248,15 @@ Usage::
 	ceph df {detail}
 
 
+features
```

Comment:
Oh i see.  In that case I wonder if we need a section elsewhere in the docs describing these two commands along with the background and how they work together.  And then link to that.   ANd keep a more concise description here.